lib/navigation.rb

```diff
@@ -29,40 +20,47 @@ def link_to_upper_page(text, attributes = {})
   attributes[:class] = "upper"
   attributes[:title] = "Back to #{upper} page"
 
-  link_to(text + " " + upper + " " + text, target, attributes)
+  link_to("&orarr; " + upper, target, attributes)
```

Comment:
Is there a reason to use HTML entity instead of the unicode symbol directly?

Reply:
Just nostalgia 🎱

Reply:
Markdown will also replace (c) with copy sign btw